slugify/slugify.py

```diff
@@ -23,8 +23,15 @@
 REMOVE_REXP = re.compile('-{2,}')
 
 
-def smart_truncate(string, max_length=0, word_boundaries=False, separator=' '):
-    """ Truncate a string """
+def smart_truncate(string, max_length=0, word_boundaries=False, save_order=False, separator=' '):
```

Comment:
save_order should come last as a good general practice, even though this function is not external.